src/lib/Scenes/Artwork/Components/CommercialEditionSetInformation.tsx

```diff
@@ -0,0 +1,126 @@
+import { Box, color, Flex, Sans, Spacer } from "@artsy/palette"
+import { CommercialEditionSetInformation_artwork } from "__generated__/CommercialEditionSetInformation_artwork.graphql"
+import React from "react"
+import { TouchableWithoutFeedback } from "react-native"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components/native"
+import { CommercialPartnerInformationFragmentContainer as CommercialPartnerInformation } from "./CommercialPartnerInformation"
+
+type EditionSet = CommercialEditionSetInformation_artwork["editionSets"][0]
+
+interface Props {
+  artwork: CommercialEditionSetInformation_artwork
+  setEditionSetId: (editionSetID: string) => void
+}
+
+interface State {
+  selectedEdition: EditionSet
+}
+
+export class CommercialEditionSetInformation extends React.Component<Props, State> {
+  state = {
+    selectedEdition: this.props.artwork.editionSets[0],
```

Comment:
Is there any chance of this erroring if `this.props.artwork.editionSets` is `undefined`?

Reply:
Good call fixing this.